jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenLayerConfigurationsTest.java

```diff
@@ -194,132 +201,6 @@ public void testGetForProject_nonDefaultAppRoot() throws IOException {
-    AbsoluteUnixPath basePath = AbsoluteUnixPath.get("/");
-
-    Predicate<Path> nameIsTarget = path -> "target".equals(path.getFileName().toString());
-    MavenLayerConfigurations.addFilesToLayer(sourceRoot, nameIsTarget, basePath, fileToLayerAdder);
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("sub/target"), basePath.resolve("sub/target"));
-    Mockito.verifyNoMoreInteractions(fileToLayerAdder);
-  }
-
-  @Test
-  public void testAddFilesToLayer_emptyDirectoryForced() throws IOException {
-    temporaryFolder.newFolder("sub", "leaf");
-
-    Path sourceRoot = temporaryFolder.getRoot().toPath();
-    AbsoluteUnixPath basePath = AbsoluteUnixPath.get("/path/in/container");
-
-    MavenLayerConfigurations.addFilesToLayer(sourceRoot, path -> false, basePath, fileToLayerAdder);
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("sub/leaf"), basePath.resolve("sub/leaf"));
-    Mockito.verifyNoMoreInteractions(fileToLayerAdder);
-  }
-
-  @Test
-  public void testAddFilesToLayer_fileAsSource() throws IOException {
-    Path sourceFile = temporaryFolder.newFile("foo").toPath();
-
-    AbsoluteUnixPath basePath = AbsoluteUnixPath.get("/");
-    try {
-      MavenLayerConfigurations.addFilesToLayer(
-          sourceFile, path -> true, basePath, fileToLayerAdder);
-    } catch (NotDirectoryException ex) {
-      Assert.assertThat(ex.getMessage(), CoreMatchers.containsString("foo is not a directory"));
-    }
-  }
-
-  @Test
-  public void testAddFilesToLayer_complex() throws IOException {
-    temporaryFolder.newFile("A.class");
-    temporaryFolder.newFile("B.java");
-    temporaryFolder.newFolder("example", "dir");
-    temporaryFolder.newFile("example/dir/C.class");
-    temporaryFolder.newFile("example/C.class");
-    temporaryFolder.newFolder("test", "resources", "leaf");
-    temporaryFolder.newFile("test/resources/D.java");
-    temporaryFolder.newFile("test/D.class");
-
-    Path sourceRoot = temporaryFolder.getRoot().toPath();
-    AbsoluteUnixPath basePath = AbsoluteUnixPath.get("/base");
-
-    Predicate<Path> isClassFile = path -> path.getFileName().toString().endsWith(".class");
-
-    MavenLayerConfigurations.addFilesToLayer(sourceRoot, isClassFile, basePath, fileToLayerAdder);
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("A.class"), basePath.resolve("A.class"));
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("example/dir/C.class"), basePath.resolve("example/dir/C.class"));
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("example/C.class"), basePath.resolve("example/C.class"));
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("test/resources/leaf"), basePath.resolve("test/resources/leaf"));
-    Mockito.verify(fileToLayerAdder)
-        .add(sourceRoot.resolve("test/D.class"), basePath.resolve("test/D.class"));
-    Mockito.verifyNoMoreInteractions(fileToLayerAdder);
-  }
-
```

Comment:
That's true, but I think it's also meaningful to have the tests, since `JavaLayerConfigurationsHelperTest` only tests the `JavaLayerConfigurationsHelper.fromExplodedWar()` method, while here, they test `MavenLayerConfigurations.getForProject()` at a higher level, so they test things at different levels.